drivers/android/process.rs

```diff
@@ -614,12 +614,12 @@ impl Process {
         for _ in 0..page_count {
             let page = Pages::<0>::new()?;
             page.insert_page(vma, address)?;
-            pages.push(page);
+            pages.try_push(page)?;
             address += 1 << bindings::PAGE_SHIFT;
         }
 
         // TODO: This allocates memory.
```

Comment:
Nit: this `TODO` was to mark a place we'd have to switch to a try function. Since you're making this change, it would be better to also remove the `TODO`.

Don't worry about it if it's too much work though. I can remove it myself alter.

Reply:
I wondered about that, but since it said "allocates memory", I thought you were referring to some optimization instead. I will remove it, it is nothing.